CRM/Core/Payment.php

```diff
@@ -589,7 +589,7 @@ public function getText($context, $params) {
         return '';
 
       case 'contributionPageContinueText':
-        return ts('Click the <strong>Continue</strong> button to proceed with the payment.');
+        return ts('Click the <strong>Make Contribution</strong> button to proceed with the payment.');
```

Comment:
How about doing the same `if` block as in the button text? It duplicates some logic, but they are called right after each other, from the same place, and the params are the same.

```php
        if ($params['amount'] <= 0.0 || (int) $this->_paymentProcessor['billing_mode'] === 4) {
          return ts('Click the <strong>Continue</strong> button to proceed with the payment.');
        }
        if ($params['is_payment_to_existing']) {
          return ts('Click the <strong>Make Payment</strong> button to proceed with the payment.');
        }
        return ts('Click the <strong>Make Contribution</strong> button to proceed with the payment.');
```